app/views/manuals/_content_for_head_tag.html.erb

```diff
@@ -2,6 +2,15 @@
   <% if description.present? %>
     <meta name='description' content='<%= description %>' />
   <% end %>
+
+  <%= render 'govuk_publishing_components/components/machine_readable_metadata',
+    schema: :article,
+    content_item: @content_store_manual,
```

Comment:
What's the reasoning behind passing the manual content item rather than the section?

Reply:
Ow, I missed that. I'll take a look.

Reply:
@sihugh updated to be the section instead of manual. Thanks!